v3-docs/v3-migration-docs/breaking-changes/call-x-callAsync.md

```diff
@@ -1,29 +1,26 @@
-# Meteor.call x Meteor.callAsync
+# Can I still use Meteor.call?
 
+You can, but we recommend you use it only to call the sync method.
 
+In fact, we log a warning message if you use `Meteor.call` to call an async method.
 
-::: tip
+If you use `Meteor.call` with async methods it can lead to unexpected behavior.
 
-It is recommened to use `Meteor.callAsync` instead of `Meteor.call` because of how our
-async API works. `Meteor.call` is still available but it is not recommended to use it, it
-can lead to unexpected behavior.
+`Meteor.callAsync` is the standard for calling async methods.
```

Comment:
It might be more accurate as something like:

```
You can, but we recommend you use it only to call methods that do not have a method stub, or when the method stub is synchronous.

In fact, we log a warning message if you use `Meteor.call` to call a method with an async method stub since it can lead to unexpected behavior.

 `Meteor.callAsync` is the standard for calling methods and supports any method, including those that have an async method stub.
```

It might also be good to have a reminder of what a method stub is.